### ISSUE TITLE
Dark/light toggle missing accessibility name

### ISSUE THREAD

**Issue by cjmabry** [With-the-Ranks/withtheranks-astro#20]
The theme toggle button doesn't have an accessible name, posing problems for screen readers.

From https://pagespeed.web.dev/analysis/https-with-the-ranks-github-io-withtheranks-astro/mzkquydov4?form_factor=desktop
